client/blocks/blog-stickers/index.jsx

```diff
@@ -13,11 +12,10 @@ import QueryBlogStickers from 'components/data/query-blog-stickers';
 import { getReaderTeams, getBlogStickers } from 'state/selectors';
 import BlogStickersList from 'blocks/blog-stickers/list';
 import InfoPopover from 'components/info-popover';
+import { isAutomatticTeamMember } from 'reader/lib/teams';
 
 const BlogStickers = ( { blogId, teams, stickers } ) => {
-	// If the user isn't in the a8c team, don't show the feature
-	const isTeamMember = !! find( teams, [ 'slug', 'a8c' ] );
-
+	const isTeamMember = isAutomatticTeamMember( teams );
 	if ( teams && ! isTeamMember ) {
```

Comment:
whats the point of the `teams` check here? if the teams endpoint starts failing, would we attempt to show users blogstickers?